Twoliter.toml

```diff
@@ -0,0 +1,8 @@
+schema-version = 1
+project-name = "bottlerocket"
+project-version = "v1.15.0"
```

Comment:
I'm a bit wary of having one more place to manage the version, in addition to `Release.toml`, but perhaps it's OK for now if `Release.toml` eventually gets deleted.